src/outagesim/backup_reliability.jl

```diff
@@ -169,38 +172,38 @@ function get_maximum_generation(batt_kw::Real, gen_capacity::Real, bin_size::Rea
     generator_prod = zeros(M, N)
     for i in 1:M
        max_battery_discharge[i, :] = fill(min(batt_kw, (i-1)*bin_size*batt_discharge_efficiency), N)
-       generator_prod[i, :] = generator_output(num_generators, gen_capacity)
+       generator_prod[i, :] = generator_output(num_generators, gen_capacity_kw)
     end
     
     return generator_prod .+ max_battery_discharge
 end
 
 """
-    battery_bin_shift(excess_generation::Vector, bin_size::Real, batt_kw::Real, batt_charge_efficiency::Real, batt_discharge_efficiency::Real)
+    battery_bin_shift(excess_generation_kw::Vector, bin_size::Real, batt_kw::Real, batt_charge_efficiency::Real, batt_discharge_efficiency::Real)
 
 Return a vector of number of bins battery is shifted by
```

Comment:
shifted by...?

Reply:
@sericson0 please address

Reply:
Just added additional description. Let me know if it needs additional text to address

Reply:
Sorry I didn't notice this before, but why does this produce a vector instead of the multiple generator outputs being summed and the battery shifted by the that total?